workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs

```diff
@@ -7,41 +7,47 @@
 using Improbable.Gdk.PlayerLifecycle;
 using Improbable.Gdk.StandardTypes;
 using Improbable.PlayerLifecycle;
+using UnityEngine;
 
 namespace Fps
 {
     public static class FpsEntityTemplates
     {
-        // TODO Add iOS client here
-        private static readonly List<string> AllWorkerAttributes =
-            new List<string> { WorkerUtils.UnityGameLogic, WorkerUtils.UnityClient, WorkerUtils.SimulatedPlayer, WorkerUtils.AndroidClient };
-
         public static EntityTemplate Spawner()
         {
-            const string gameLogic = WorkerUtils.UnityGameLogic;
-
-            return EntityBuilder.Begin()
-                .AddPosition(0, 0, 0, gameLogic)
-                .AddMetadata("PlayerCreator", gameLogic)
-                .SetPersistence(true)
-                .SetReadAcl(gameLogic)
-                .AddComponent(PlayerCreator.Component.CreateSchemaComponentData(), gameLogic)
-                .Build();
+            var position = new Position.Snapshot { Coords = new Vector3().ToSpatialCoordinates() };
```

Comment:
An empty constructor has the same result. Although this is more explicit

Reply:
will change it to an empty constructor

Reply:
There is also a `Vector3.zero` which will be more explicit.